functions/FitProgram/criterionGauss_diffrho.m

```diff
@@ -25,7 +25,7 @@
 beta = thetanonlin(1:2);
 rho = thetanonlin(end);
 
-R = sqrt((Xreshape - beta(1)).^2 + (Yreshape - beta(2)).^2);
+R = (Xreshape - beta(1)).^2 + (Yreshape - beta(2)).^2;
```

Comment:
makes sense, perhaps consider renaming this to R2, in analogy with rho2, rho3 variables?